kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/LogReplayEngineMetricsSuite.scala

```diff
@@ -412,23 +412,23 @@ class LogReplayEngineMetricsSuite extends AnyFunSuite with TestUtils {
       buildTableWithCrc(path)
         Seq(10L, 11L, 5L, 6L).foreach { version =>
         assert(
-            Files.deleteIfExists(
-              new File(
-                FileNames.checksumFile(new Path(f"$path/_delta_log"), version).toString
-              ).toPath
-            )
+          Files.deleteIfExists(
+            new File(
+              FileNames.checksumFile(new Path(f"$path/_delta_log"), version).toString
+            ).toPath
           )
+        )
       }
 
       loadPandMCheckMetrics(
         Table.forPath(engine, path)
           .getLatestSnapshot(engine).getSchema(),
         engine,
-          // 10.checkpoint found, so use it and combined with 11.crc
-          expJsonVersionsRead = Seq(11),
-          expParquetVersionsRead = Seq(10),
-          expParquetReadSetSizes = Seq(1),
-          expChecksumReadSet = Seq(11)
+        // 10.checkpoint found, so use it and combined with 11.crc
+        expJsonVersionsRead = Seq(11),
+        expParquetVersionsRead = Seq(10),
+        expParquetReadSetSizes = Seq(1),
+        expChecksumReadSet = Nil
         )
```

Comment:
I think the indentation here is all messaged up (not your fault). I think the bracket here needs to be 2 spaces to the left.

I also think the next `loadPandMCheckMetrics` is also wrong -- that entire block needs to go to the left.